build/Uno.UI.Build.csproj

```diff
@@ -44,7 +44,7 @@
 
 	<ItemGroup Condition="$([MSBuild]::IsOsPlatform(Windows))">
 		<PackageReference Include="GitVersionTask">
-			<Version>4.0.0</Version>
+			<Version>5.1.2</Version>
 		</PackageReference>
```

Comment:
This will cause GitVersion to run twice, shouldn't we remove it?

Reply:
True. Will take a look for the next round